apps/omg_child_chain/lib/omg_child_chain/block_queue.ex

```diff
@@ -80,15 +83,13 @@ defmodule OMG.ChildChain.BlockQueue do
       {:ok, state} =
         with {:ok, _state} = result <-
                Core.new(
-                 mined_child_block_num: mined_num,
-                 known_hashes: Enum.zip(range, known_hashes),
-                 top_mined_hash: top_mined_hash,
-                 parent_height: parent_height,
+                 mined_num,
+                 Enum.zip(range, known_hashes),
+                 top_mined_hash,
+                 parent_height,
                  child_block_interval: child_block_interval,
-                 chain_start_parent_height: parent_start,
                  block_submit_every_nth: Application.fetch_env!(:omg_child_chain, :block_submit_every_nth),
```

Comment:
A good pattern coming from OTP is that children never directly access configuration. Configuration values are always passed top to bottom. Application process to Supervisor to it's children.